tf2onnx/tf_loader.py

```diff
@@ -231,29 +231,32 @@ def from_checkpoint(model_path, input_names, output_names):
     return frozen_graph, input_names, output_names
 
 
-def _from_saved_model_v1(sess, model_path, input_names, output_names, tag, signatures):
+def _from_saved_model_v1(sess, model_path, input_names, output_names, tag, signature_names):
     """Load tensorflow graph from saved_model."""
 
     wrn_no_tag = "'--tag' not specified for saved_model. Using --tag serve"
-    wrn_empty_tag = "'--tag' value is empty string. Using tag =[[]]"
+    wrn_empty_tag = "'--tag' value is empty string. Using tags = []"
+    wrn_empty_sig = "'--signature_def' not provided. Using all signatures."
 
     if tag is None:
         tag = [tf.saved_model.tag_constants.SERVING]
         logger.warning(wrn_no_tag)
 
+    if not signature_names:
+        logger.warning(wrn_empty_sig)
+
     if tag == '':
-        tag = [[]]
```

Comment:
TF1 complains that list is an unhashable type.  We actually want to specify an empty set, which is `set([])` not `set([[]])`